src/bokeh/core/property/visual.py

```diff
@@ -170,7 +170,8 @@ def transform(self, value):
         if isinstance(value, np.ndarray):
             value = PIL.Image.fromarray(value)
 
-        if isinstance(value, str) and value.startswith("data:image/"):
+        if isinstance(value, str) and (
+            value.startswith("data:image/") or value in enums.ToolIcon):
             return value
 
         if isinstance(value, str):
```

Comment:
I think that at this point we should remove this deprecated branch and then the above condition would be reduced to just  `isinstance(value, str)`.

Reply:
@mattpap Thanks for your recommendation. I'll try to get in the change.